### PR TITLE
Fixed all the paths in README and in CONTRIBUTING to be relative.

### PR DESCRIPTION
**Pull request by hpoom**
Fixed all the paths in README and in CONTRIBUTING to be relative. Will no longer be an issue when forking.

@jodiedoubleday can you :+1:  this and I will then merge.

### CODE REVIEW

**Comment by jodiedoubleday** [comment]
I'll :+1: when you change back to 2 spaces :wink:

**Comment by hpoom** [comment]
Doh. Thats your fault. No `.editorconfig` file so Brackets just changed it all to be correct :smile:

**Comment by hpoom** [comment]
Finally it all looks good @jodiedoubleday 

Please :+1:  and we can get this merged :smile:

File: README.md [hunk 1]
```diff
@@ -48,9 +48,9 @@ npm install
 Troubleshooting
 ---
 If you have any problems installing the ui-toolkit then please view the
-[troubleshooting guide](https://github.com/holidayextras/ui-toolkit/blob/master/TROUBLESHOOTING.md) which contains common errors and
+[troubleshooting guide](/blob/master/TROUBLESHOOTING.md) which contains common errors and
```
Comment (jodiedoubleday) [inline README.md]: I think from one markdown file to another you can just call the file `TROUBLESHOOTING` and it will go there

**Comment by jodiedoubleday** [comment]
:+1:

**Pull request merged**